### DISCUSSION TITLE
I fully support Ukraine and Bless You for making this INCREDIBLE software

### DISCUSSION THREAD

**Discussion by Mandoon14** [Tyrrrz/LightBulb#243] [General]
First off this software is better than CareUEyes, a paid-for software, I intend to donate to you at the end of the month when I get money as I don't have enough right now (I will bookmark your page etc to remind me). I have already (of course) donated what little I have both in physical items (mainly baby food and products) and money to support your country and its people. Be safe and I wish you so much goodness, support and kindness in the future. Peace be with you at all times! I'm agnostic but just in case, God Bless You! And Happy Easter!

**Comment by Tyrrrz** [comment]
Thank you very much 🙂 ❤️ 
Please don't feel bad if you can't donate. Your sentiment alone is already greatly appreciated.

PS: it's [`Ukraine`, not `the Ukraine`](https://en.wikipedia.org/wiki/Name_of_Ukraine)

**Reply by Mandoon14** [reply]
I will donate as the software is well worth it and it's going to someone who needs it more than me. You're program rocks and I thank you very much, man!

You take care of yourself now you hear.

**Reply by Mandoon14** [reply]
Sorry, I'm from "the" UK (you can see why I got it wrong) so although not illiterate I make mistakes. I will remember it's Ukraine.

PS: Sorry btw

**Comment by IntendedPun** [comment]
Another "Thank you so much for making this application!" comment here! I've tried a few different ones and LightBulb is hands down the best one. Most apps don't offer this level of customisation, like manual sunset/sunrise time and transition control.

I live in Finland so I can sympathise with your situation down in the south. Stay safe out there!
